templates/project-ts/src/index.ts

```diff
@@ -18,9 +18,15 @@ import {
 export class Add extends SmartContract {
   @state(Field) num = State<Field>();
 
-  // initialization
   deploy(args: DeployArgs) {
```

Comment:
Do we need this deploy method still?

Reply:
I think it's helpful to have it here just to expose that this function exists for people first trying out SnarkyJS. In other examples, we should try to optimize it out if it's not needed